client.js

```diff
@@ -162,6 +163,7 @@ const sendGeminiRequest = (pathOrUrl, opt, done) => {
 		hostname: target.hostname || 'localhost',
 		port: target.port || DEFAULT_PORT,
 		tlsOpt,
+		verifyAlpnId,
```

Comment:
I designed the API so that `sendGeminiRequest` can be used by itself, without messing with convenient defaults. `_request` expects `verifyAlpnId` to be a function, but `sendGeminiRequest` will no pass `undefined` in.

Please only pass this function in if a) `"verifyAlpnId` is a member of `opt`, or b) if the value is not `undefined` or `null`. Up to you.